modules/network-observability-enriched-flows.adoc

```diff
@@ -37,11 +38,29 @@ spec:
         targetHost: "ipfix-collector.ipfix.svc.cluster.local"
         targetPort: 4739
         transport: tcp or udp           <4>
-
-
+ -  type: OpenTelemetry                 <3>
+      openTelemetry:
+        targetHost: my-otelcol-collector-headless.otlp.svc
+        targetPort: 4317
+        type: grpc                      <5>
+        logs:                           <6>
+          enable: true
+        metrics:                        <7>
+          enable: true
+          prefix: netobserv
+          pushTimeInterval: 20s         <8>
+          expiryTime: 2m
+   #    fieldsMapping:                  <9>
+   #      input: SrcAddr
+   #      output: source.address
 ----
 <1> The Network Observability Operator exports all flows to the configured Kafka topic.
 <2> You can encrypt all communications to and from Kafka with SSL/TLS or mTLS. When enabled, the Kafka CA certificate must be available as a ConfigMap or a Secret, both in the namespace where the `flowlogs-pipeline` processor component is deployed (default: netobserv). It must be referenced with `spec.exporters.tls.caCert`. When using mTLS, client secrets must be available in these namespaces as well (they can be generated for instance using the AMQ Streams User Operator) and referenced with `spec.exporters.tls.userCert`.
-<3> You can export flows to IPFIX instead of or in conjunction with exporting flows to Kafka.
+<3> You can export flows to IPFIX, OpenTelemetry, and Kafka individually or concurrently. 
```

Comment:
I guess this already existed this way before this PR, but it seems odd that the first item in the YAML with a callout has a callout of 3. Is it possible to bump this up to 1, so that there's numerical order in the YAML (at least whenever a unique labelled item appears in the file)?

Reply:
That's a good idea. I think it ended up this way when I added IPFIX to the mix, because previously this was just a topic about Kafka exporting. Its since become IPFIX, Kafka, and now OTEL. So yeah I think thats a good point to  rearrange these callouts to make them more numerically ordered.